src/packet.proto

```diff
@@ -2,6 +2,23 @@ syntax = "proto3";
 
 package helium;
 
+message gps_info {
+    float horizontal_accuracy = 1; // meters this will take into account h3 precision and distance from asserted location
+    float vertical_accuracy = 2; // meters
+    int32 altitude = 3; // meters
+    uint64 time_accuracy = 4; //nanoseconds
+}
+
+// https://www.eecis.udel.edu/~mills/ntp/html/ntpq.html#system
+message ntp_info {
+    bool synced = 1;
```

Comment:
So this means that it has synced with some ntp server? Don't see it in the ntp system link above, just wondering.